dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/MethodSymbol.java

```diff
@@ -65,18 +65,16 @@ public boolean equals(Object other) {
     if (other == null || getClass() != other.getClass()) {
       return false;
     }
-    if (!super.equals(other)) {
-      return false;
-    }
     MethodSymbol that = (MethodSymbol) other;
     return isInterfaceMethod == that.isInterfaceMethod
         && name.equals(that.name)
-        && descriptor.equals(that.descriptor);
+        && descriptor.equals(that.descriptor)
+        && this.getClassBinaryName().equals(that.getClassBinaryName());
```

Comment:
Checking the equality of class name here is more clear than `super.equals`. Nice.